src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -2053,24 +2057,69 @@ UpdateShardPlacement(PlacementUpdateEvent *placementUpdateEvent,
  * ExecuteCriticalCommandInSeparateTransaction runs a command in a separate
  * transaction that is commited right away. This is useful for things that you
  * don't want to rollback when the current transaction is rolled back.
+ * Set true to 'useExclusiveTransactionBlock' to initiate a BEGIN and COMMIT statements.
  */
 void
-ExecuteRebalancerCommandInSeparateTransaction(char *command)
+ExecuteRebalancerCommandInSeparateTransaction(char *command, bool
+											  useExclusiveTransactionBlock)
 {
 	int connectionFlag = FORCE_NEW_CONNECTION;
 	MultiConnection *connection = GetNodeConnection(connectionFlag, LocalHostName,
 													PostPortNumber);
 	StringInfo setApplicationName = makeStringInfo();
```

Comment:
Old variable name doesn't make sense anymore
```suggestion
	StringInfo setStatements = makeStringInfo();
```